db/migrate/20180424141617_azure_backslash_to_forward_slash.rb

```diff
@@ -26,6 +26,7 @@ class EventStream < ActiveRecord::Base
   def up
     say_with_time("Updating Azure VM ems_ref, uid_ems and description") do
       Vm.where(:type => "ManageIQ::Providers::Azure::CloudManager::Vm")
+        .where.not(:ems_ref => nil)
```

Comment:
The `EventStream.vm_ems_ref` being nil doesn't mean that the VM's ems_ref is nil...just means that the event wasn't about a VM.

I don't think it is possible (or at least it really shouldn't be) to have a vm with a nil ems_ref.

Reply:
@agrare so then we should just stick with the original PR for event streams I think.